docs/cli/kops_edit_cluster.md

```diff
@@ -7,7 +7,7 @@ Edit cluster.
 
 Edit a cluster configuration. 
 
-This command changes the cluster cloud specification in the registry. 
+This command changes the cluster cloud configuration in the registry. 
```

Comment:
I'd say either "cluster specification" or "desired cluster configuration".

And I'm not even sure we need "in the registry", but that's just my opinion.